ui/app/AppLayouts/Profile/Sections/Ens/TermsAndConditions.qml

```diff
@@ -23,6 +22,35 @@ Item {
         font.pixelSize: 20
     }
 
+    ModalPopup {
+        id: transactionDialog
+        title: qsTr("TODO: replace this for the transaction dialog")
+
+        Input {
+            id: passwd
+            placeholderText: "Password"
+            anchors.top: parent.textToCopy
+            anchors.topMargin: 24
+            anchors.left: parent.left
+            anchors.leftMargin: 24
```

Comment:
```suggestion
            anchors.leftMargin: Style.current.bigPadding
```

Reply:
This one is not really necessary. This input field is part of a ModalPopup that needs to be replaced for a proper transaction dialog. I created it just to have some way to input the password but it should not be used in the final version